packages/teleport/src/DesktopSession/WarningDropdown.tsx

```diff
@@ -0,0 +1,148 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { useRef, useState, useEffect } from 'react';
+import { Text, Flex, Button, Card, ButtonIcon } from 'design';
+import styled from 'styled-components';
+import { Notification, NotificationItem } from 'shared/components/Notification';
+import { Warning, Close } from 'design/Icon';
+import { orange } from 'design/theme/palette';
+
+export function WarningDropdown(props: Props) {
+  const { warnings, onRemoveWarning } = props;
```

Comment:
personal preference but I prefer to have the destructuring in the function declaration. 

```ts
export function WarningDropdown({ warnings, onRemoveWarning }: Props) {
```